### PR TITLE
BONNIE-757: adding 'component' attribute to 2 datatypes

### PR DESCRIPTION
**Pull request by c-monkey**
'Physical Exam, Performed' and 'Procedure, Performed'

See https://github.com/projecttacoma/bonnie/pull/807 which needs this PR to work.

### CODE REVIEW

File: app/assets/javascripts/datatypes/physicalexam.js.coffee [hunk 1]
```diff
@@ -79,6 +79,7 @@ class CQL_QDM.PhysicalExamPerformed extends CQL_QDM.QDMDatatype
       @_relevantPeriodHigh = CQL_QDM.Helpers.infinityDateTime()
     if @entry.values? && @entry.values.length > 0
       @_result = @entry.values?[0]
+    @_component = @entry.components
```
Comment (losborne) [inline app/assets/javascripts/datatypes/physicalexam.js.coffee]: This should be @_components = @entry.components to match my pluralization PR (which hasn't been merged yet).

File: app/assets/javascripts/datatypes/physicalexam.js.coffee [hunk 2]
```diff
@@ -125,6 +126,16 @@ class CQL_QDM.PhysicalExamPerformed extends CQL_QDM.QDMDatatype
   result: ->
     CQL_QDM.Helpers.formatResult(@_result)
 
+  ###
+  @returns {Array}
+  ###
+  component: ->
```
Comment (losborne) [inline app/assets/javascripts/datatypes/physicalexam.js.coffee]: This should similarly be changed to components.

File: app/assets/javascripts/datatypes/physicalexam.js.coffee [hunk 3]
```diff
@@ -125,6 +126,16 @@ class CQL_QDM.PhysicalExamPerformed extends CQL_QDM.QDMDatatype
...
+    components = []
+    if @_component
```
Comment (losborne) [inline app/assets/javascripts/datatypes/physicalexam.js.coffee]: This and the next line should be @_components, similar to above.

File: app/assets/javascripts/datatypes/procedure.js.coffee [hunk 4]
```diff
@@ -102,6 +102,7 @@ class CQL_QDM.ProcedurePerformed extends CQL_QDM.QDMDatatype
     if @entry.values? && @entry.values.length > 0
       @_result = @entry.values?[0]
     @_status = @entry.status
+    @_component = @entry.components
```
Comment (losborne) [inline app/assets/javascripts/datatypes/procedure.js.coffee]: Same comment as above.

**Comment by c-monkey** [comment]
I have made your suggested changes, Luke

**Pull request merged**